configuration/transitions.md

```diff
@@ -39,6 +40,30 @@ In this example the `d` transition will not be applied, but the other three will
 
 Guides for how to setup specific transitions.
 
+### multi_form_alerts
+
+Full documentation in [kanso.json](https://github.com/medic/medic-webapp/blob/master/kanso.json) or in the Dashboard app_settings page.
```

Comment:
Kanso and Dashboard are going away. I think we need to add full field level documentation to this section and then remove this line.